src/components/Layout/Layout.tsx

```diff
@@ -0,0 +1,9 @@
+import styles from "./Layout.module.scss";
+
+type ChildrenType = {
+	children: JSX.Element;
```

Comment:
Czasem typujesz children jako JSX.Element, a czasem ReactNode. Ważne jest pisać kod w jednym stylu. Tym bardziej wtedy, kiedy pisze się go samemu. Seniorzy rekrutując również zwracają i na to uwagę.